linked_list.py

```diff
@@ -0,0 +1,60 @@
+
+#import numpy as np
+
+# insertAt
+# deleteAt
+# first
+# last
+
+class Node:
+    def __init__(self, data=None):
```

Comment:
why is data None by default? If the consumer wants to send None, it can pass ; than you deciding its None?

Reply:
Done